PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/serviceexplorer/azure/appservice/SSHTerminalManager.java

```
@@ -0,0 +1,223 @@
+            Messages.showWarningDialog(SSH_INTO_WEB_APP_ERROR_MESSAGE_NOT_SUPPORT_DOCKER, SSH_INTO_WEB_APP_ERROR_DIALOG_TITLE);
+            return false;
+        }
+        return true;
+    }
+
+    /**
+     * create remote connection to remote web app container.
+     *
+     * @param parameters
+     * @return
+     */
+    public CreateRemoteConnectionOutput executeAzCreateRemoteConnectionAndGetOutput(final String[] parameters) {
+        CreateRemoteConnectionOutput connectionInfo = new CreateRemoteConnectionOutput();
+        CommandUtils.CommendExecOutput commendExecOutput = null;
+        try {
+            commendExecOutput = AzureCliUtils.executeCommandAndGetOutputWithCompleteKeyWord(parameters,
+                    AzureCliUtils.CLI_COMMAND_REMOTE_CONNECTION_EXEC_SUCCESS_KEY_WORDS, AzureCliUtils.CLI_COMMAND_REMOTE_CONNECTION_EXEC_FAILED_KEY_WORDS);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        connectionInfo.setOutputMessage(commendExecOutput.getOutputMessage());
+        connectionInfo.setSuccess(commendExecOutput.getSuccess());
+        if (commendExecOutput.getSuccess()) {
+            String username = PatternUtils.parseWordByPatternAndPrefix(commendExecOutput.getOutputMessage(), PatternUtils.PATTERN_WHOLE_WORD, "username: ");
+            if (StringUtils.isNotBlank(username)) {
+                connectionInfo.setUsername(username);
+            }
+            String port = PatternUtils.parseWordByPatternAndPrefix(commendExecOutput.getOutputMessage(), PatternUtils.PATTERN_WHOLE_NUMBER_PORT, "port: ");
+            if (StringUtils.isNotBlank(port)) {
+                connectionInfo.setPort(port);
+            }
+            String password = PatternUtils.parseWordByPatternAndPrefix(commendExecOutput.getOutputMessage(), PatternUtils.PATTERN_WHOLE_WORD, "password: ");
+            if (StringUtils.isNotBlank(password)) {
+                connectionInfo.setPassword(password);
+            }
+        } else {
+
```

throw Exception rather than an empty else